src/components/ContainerItems/containerItems.tsx

```diff
@@ -0,0 +1,25 @@
+import './style.css';
+
+interface ContainerItemsProps {
+  title: string;
+  children: React.ReactNode;
+  backgroundBege: boolean;
+}
+
+const ContainerItems = ({ title, children, backgroundBege }: ContainerItemsProps) => {
+  return (
+    <>
+      <div className={`${backgroundBege ? 'container-carousel-event' : 'container-carousel-items'}`}>
+        <div className={`${backgroundBege ? 'content-title-event' : 'content-title'}`}>
+          <span className={`${backgroundBege ? 'title-carousel-event' : 'title-carousel-items'}`}>
+            {title} {'>'}
+          </span>
+        </div>
+      </div>
+
+      <div className={`${backgroundBege ? 'space-carousel-events' : 'space-carousel-items'}`}>{children}</div>
+    </>
+  );
+};
```

Comment:
Teria que padronizar alguns classNames, mas o que acha?

```suggestion
const ContainerItems = ({ title, children, backgroundBege }: ContainerItemsProps) => {
  
  const getClassName = (classNameBase: string) {
      if (backgroundBege) return classNameBase + '-events';
      return classNameBase + '-items'
  }
  
  return (
    <>
      <div className={getClassName('container-carousel')}}>
        <div className={getClassName('content-title')}>
          <span className={getClassName('title-carousel')}>
            {title} {'>'}
          </span>
        </div>
      </div>

      <div className={getClassName('space-carousel')}>{children}</div>
    </>
  );
};
```